tests/js/spec/views/performance/landing.spec.jsx

```diff
@@ -475,66 +474,6 @@ describe('Performance > Landing', function () {
     );
   });
 
-  it('Vitals cards are not shown with overview feature without frontend platform', async function () {
```

Comment:
Should these tests be kept? Looks like they're testing for the display mode based on the project platform.